src/lib/parameters/param_translation.cpp

```diff
@@ -250,5 +250,16 @@ bool param_modify_on_import(bson_node_t node)
 		}
 	}
 
+	// 2022-11-09: translate VT_F_TRANS_THR/VT_PSHER_RMP_DT -> VT_PSHER_SLEW)
+	{
+		if (strcmp("VT_PSHER_RMP_DT", node->name) == 0) {
+			strcpy(node->name, "VT_PSHER_SLEW");
+			float _param_vt_f_trans_thr = param_find("VT_F_TRANS_THR");
+			node->d = _param_vt_f_trans_thr / node->d;
```

Comment:
This fails to build because of implicit double-promotion. I think if you replace `float` with `double` it should work.